source/api_reference/version-history.html.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Version history
+weight: 0
+---
+# Version history
+
+## Version 1
+
+Version 1 can be accessed using either of these base URLs:
+
+- `https://{register-name}.register.gov.uk/v1/`
```

Comment:
We could suggest `/v1` is the preferred way (I imagine the other one will end up being a redirection)

Reply:
It's implemented so that /v1/ is the redirect. I don't think it matters much though, and the plan is to remove it in a few months anyway.